test/unit/spec/EnrollChoices_spec.js

```diff
@@ -305,6 +315,21 @@ function (Okta, OktaAuth, Util, EnrollChoicesForm, Beacon, Expect, Router,
             );
           });
         });
+        itp('does not have link to skip set up during grace period when all factors are optional none enrolled', function () {
```

Comment:
similar comment for the name of these tests. Widget does not really know anything about a grace period, all we know is that the skip link is present even if some required factors are not enrolled yet

Reply:
👍 removing grace period for all tests